main/admin-panel/src/jobs.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  List,
+  Datagrid,
+  EditButton,
+  Create,
+  SimpleForm,
+  BooleanInput,
+  TextInput,
+  Edit,
+  useListContext,
+  Show,
+  SimpleShowLayout,
+  RichTextField,
+  TextField,
+  BooleanField,
+  DateField,
+} from "react-admin";
+import { RichTextInput } from "ra-input-rich-text";
+import { Button, Toolbar } from "@mui/material";
+import ChevronLeft from "@mui/icons-material/ChevronLeft";
+import ChevronRight from "@mui/icons-material/ChevronRight";
+
+const JobPagination = () => {
+  const { page, hasPreviousPage, hasNextPage, setPage } = useListContext();
+  if (!hasPreviousPage && !hasNextPage) return null;
+  return (
+    <Toolbar>
+      {hasPreviousPage && (
+        <Button
+          key="previous"
+          onClick={() => setPage(page - 1)}
+          startIcon={<ChevronLeft />}
+        >
+          Previous
+        </Button>
+      )}
+      {hasNextPage && (
+        <Button
+          key="next"
+          onClick={() => setPage(page + 1)}
+          startIcon={<ChevronRight />}
+        >
+          Next
+        </Button>
+      )}
+    </Toolbar>
+  );
+};
```

Comment:
I think that should be a generic component which could be used for all paginations therefore it should be placed in: `src/components/pagination/index.tsx`